docs/unit_conversion.rst

```diff
@@ -0,0 +1,26 @@
+.. _specviz-unit-conversion:
+
+Unit Conversion
+===============
+
+How Unit Conversion Works
+-------------------------
+
+Unit Conversion uses the `astropy.units <http://docs.astropy.org/en/stable/units/>`_ module in order to convert spectral
+axis and flux units.
+.. code-block:: python
```

Comment:
I think maybe you need a space above this directive?  Usually rst depends on blank lines to recognize things like this...